ios/Classes/PspdfkitPlatformViewImpl.swift

```diff
@@ -35,7 +35,21 @@ public class PspdfkitPlatformViewImpl: NSObject, PspdfkitWidgetControllerApi, PD
         }
         pspdfkitWidgetCallbacks?.onPageChanged(documentId:document.uid , pageIndex: Int64(pageIndex)){ _ in }
     }
-        
+    
```

Comment:
The selected line implements the method pdfViewController(_:menuForText:onPageView:appearance:suggestedMenu:) to the PspdfkitPlatformViewImpl class. This method creates and returns a custom context menu when text is selected in the PDF view. The custom menu includes a single action titled "Custom Action" that prints "Custom Action" to the console when triggered.